packs/github/actions/create_issue.py

```diff
@@ -0,0 +1,18 @@
+from lib.base import BaseGithubAction
+from github import GithubObject
+from lib.formatters import issue_to_dict
+
+__all__ = [
+    'CreateIssueAction'
+]
+
+
+class CreateIssueAction(BaseGithubAction):
+    def run(self, user, repo, title, description, assignee):
```

Comment:
Like this: https://github.com/StackStorm/st2contrib/blob/master/packs/mailgun/actions/send_email.py#L15